plugin-k8s/alumet-user.yaml

```diff
@@ -0,0 +1,41 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  annotations:
```

Comment:
I think this will raise an error

Reply:
I used this file, and no problem occurred.

Reply:
That's weird

Reply:
should i delete the **annotations** part ?